### ISSUE TITLE
default shortcut conflict with anki built-in ones

### ISSUE THREAD

**Issue by ijgnd** [kleinerpirat/anki-tooltips#1]
"Ctrl+T, ..." is used  is used for insertig mathjax or latex blocks. Your shortcut Ctrl+T doesn't seem to block/override the built in ones. In the latest version (git commit d9f1e2264) and 2.1.55 the following happens:

Ctrl+T opens your tooltip window, but then I can't directly write words starting with T or M, typing the letter "t" is then transformed to "[latex][/latex]" and the cursor is in the middle of the latex block. I have to manually delete this. (Or did I really miss that you can now remap Anki default shortcuts?)

That's strange.

&nbsp;

PS 
- Thanks for publishing this add-on as a reference. I will carefully study it. This looks very useful for me, e.g. for my [note linking add-on](https://github.com/ijgnd/anki__preview_card_edit_note_in_extra_window_linked_from_reviewer/issues/17).

**Comment by kleinerpirat** [comment]
Yep, I noticed that too. Overriding the shortcut from `editor.py` should fix this.

**Comment by kleinerpirat** [comment]
Fixed with https://github.com/kleinerpirat/anki-tooltips/commit/b9aef64734a12c93e5e8a3d735f89f5c4d0db719.

**Comment by kleinerpirat** [comment]
I was too quick to close this. If I override the shortcut with an empty function, then the empty function will block the tooltips instead of the native one, but we're in the same situation.
